src/components/SearchBar.vue

```diff
@@ -35,8 +35,25 @@
             },
             onResultSelect(event) {
                 this.map.indoors.setFloor(event.result.data.floor_id);
-                this.map.setView(event.result.location.latLng, 20);
+                this.map.setView(event.result.location.latLng, 18);
+
                 this.route({map: this.map, destination: [event.result.data.lon, event.result.data.lat, event.result.data.floor_id]});
+
+                const markerController = new window.WrldMarkerController(this.map, { poiViewsEnabled: true });
```

Comment:
I envisioned this "pop up" being our own custom component. That way we can have it be places on the side of the scrollbar and style it out how we want. But please add an image of how this looks so we can see if we should leave it as is.

Reply:
I was originally planning on leaving the poicard as a placeholder as I figure out how to point the info to the scrollbar but now I think I might have an idea